ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Program
+    {
+         static void Main(string[] args)
+        {
+            Console.WriteLine($"Ввод числа n");
+
+            int n = DoubleSquereFunction(int.Parse(Console.ReadLine()));
+
+            Console.WriteLine($"Результат функций равен = {n}");
+            Console.ReadKey();
+        }
+        static int DoubleSquereFunction(int n)
+        {
+            int x = (int)Math.Pow(n, 2);
+
+            for (int i = 1; i <= n; i++)
+            {
+                x += (n + i) * (n + i);
+            }
```

Comment:
1. Возводить в квадрат проще умножением.
2. Если в теле цикла одна строка, то `{ }` можно не использовать.
```suggestion
            int x = n * n;

            for (int i = 1; i <= n; i++)
                x += (n + i) * (n + i);
```